File: 1-9/problem 1.py
# If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
# Find the sum of all the multiples of 3 or 5 below 1000.

# use compute as it is faster
def compute():
	ans = sum(x for x in range(1000) if (x % 3 == 0 or x % 5 == 0))
	return str(ans)

print(compute())

# 233168

File: 1-9/problem 1.cpp
#include <iostream>

using namespace std;

int main() {
  int sum = 0; // Initialize the sum to 0

  // Iterate through all numbers from 1 to 1000
  for (int i = 1; i < 1000; i++) {
    // If the number is a multiple of 3 or 5, add it to the sum
    if (i % 3 == 0 || i % 5 == 0) {
      sum += i;
    }
  }

  // Print the sum
  cout << sum << endl;

  return 0;
}
